### ISSUE TITLE
PunkBuster service doesn't start when it is installed in the sandbox

### ISSUE THREAD

**Issue by ghost** [sandboxie-plus/Sandboxie#3136]
### Describe what you noticed and did

1. Download the latest version of [PunkBuster Service Installer](http://evenbalance.com/downloads/pbsvc/pbsvc.exe)
2. Launch the installer via Sandboxie
3. Get an error message:
![image](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/3b8d4395-8dd6-4ba3-a535-5521133f3994)

Tested in Standard and Application Compartment box with all presets of Admin Rights (UAC, Drop Emulate).

### How often did you encounter it so far?

Always

### Affected program

PunkBuster

### Download link

http://evenbalance.com/downloads/pbsvc/pbsvc.exe

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Expected behavior

If it is technically possible, it would be expected to launch the sandboxed PunkBuster service.

### What is your Windows edition and version?

Windows 7 x64 SP1

### In which Windows account you have this problem?

An account with UAC protection completely turned off.

### Please mention any installed security software

Microsoft Security Essentials

### What version of Sandboxie are you running?

Sandboxie Plus x64 1.10.1 / 1.10.2

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I don't remember which one).

### In which sandbox type you have this problem?

All sandbox types (I tried them all).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

**Comment by offhub** [comment]
I don't think it is possible to install PunkBuster and similar anti-cheat software in a sandbox.

See for more: https://github.com/sandboxie-plus/Sandboxie/labels/DRM%20software

**Comment by isaak654** [comment]
I don't think the lead developer has looked into anti-cheats yet.

If there is a way, no one has found it so far.

**Comment by isaak654** [comment]
The lead developer hasn't looked into anti-cheats yet.

If there is a way, no one else has confirmed it so far.

**Comment by isaak654** [comment]
Two possible attempts:
- a batch file configured like in [this article](https://www.ghacks.net/2010/04/23/run-punkbuster-services-only-when-needed/)
- use the undocumented [StartService](https://github.com/sandboxie-plus/sandboxie-docs/pull/90#pullrequestreview-1122248517) command

**Comment by isaak654** [comment]
> > * use the undocumented [StartService](https://github.com/sandboxie-plus/sandboxie-docs/pull/90#pullrequestreview-1122248517) command
> 
> Does StartService apply to OS service or a service installed in a box?

The first one, this is why it is the last choice.

**Comment by DavidXanatos** [comment]
> > * use the undocumented [StartService](https://github.com/sandboxie-plus/sandboxie-docs/pull/90#pullrequestreview-1122248517) command
> 
> Does StartService apply to OS service or a service installed in a box?

it should be able to start a service installed in a box

**Comment by isaak654** [comment]
I've just tested it and the [installer](http://evenbalance.com/downloads/pbsvc/pbsvc.exe) remains stuck at step 2 with Sandboxie Plus x64 1.10.2 as displayed [here](https://user-images.githubusercontent.com/93145383/255740543-3b8d4395-8dd6-4ba3-a535-5521133f3994.png), so I'm unable to install it at all. Both standard and application compartment sandboxes have the same behavior.
